lib/components/atoms/ListItem/ListItem.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+/**
+ *
+ * ListItem
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './ListItem.style';
+import type { Props } from './types';
+
+const ListItem = ({ className, children }: Props): Node => {
+  const classes = ['item', className].join(' ');
```

Comment:
You don't need to do this we already have a module classnames that does this class joining well